deployment/flux/l4/helm-release-l4.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
-  name: iot-edge-accelerator-release
+  name: iot-edge-l4-release
```

Comment:
nit: wonder if we should name the app here e.g. iot-edge-app1-l4-release

Reply:
Done, also for L2.